libs/vertexai/tests/integration_tests/test_chat_models.py

```diff
@@ -235,6 +244,107 @@ def test_multimodal_media_inline_base64(file_uri, mime_type) -> None:
     assert isinstance(output.content, str)
 
 
+@pytest.mark.release
+@pytest.mark.parametrize("file_uri,mime_type", multimodal_inputs)
+def test_multimodal_media_inline_base64_template(file_uri, mime_type) -> None:
+    llm = ChatVertexAI(model_name="gemini-1.5-pro-preview-0514")
```

Comment:
why preview model here? we should start getting away from it

Reply:
fair! not sure why i did it either. updated.